app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt

```diff
@@ -230,6 +230,7 @@ class LoginActivity : AppCompatActivity(), ILoginView {
     }
 
     override fun onBackPressed() {
+        super.onBackPressed()
         loginPresenter.skipLogin()
         super.onBackPressed()
```

Comment:
Why twice?

Reply:
THis happened by mistake while resolving the merge conflict. Have removed it in the update.